test/test_checkpoint.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not
+# use this file except in compliance with the License. A copy of the License
+# is located at
+#
+#     http://aws.amazon.com/apache2.0/
+# 
+# or in the "license" file accompanying this file. This file is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from math import isclose
+import mxnet as mx
+import random
+import tempfile
+
+import sockeye.data_io
+
+def generate_random_sentence(vocab_size, max_len):
+    length = random.randint(1, max_len)
+    sentence = []
+    for _ in range(length):
```

Comment:
Add a comment why +2
Also:
return [random.randint(3, vocab_size + 2) for _ in range(length)]